src/apiClient.js

```diff
@@ -4053,7 +4056,7 @@ function normalizeReturnBitrate(instance, bitrate) {
         return Promise.reject();
     }
 
-    let result = Math.round(bitrate * 0.7);
+    let result = Math.min(Math.round(bitrate * 0.7), MAX_BITRATE);
```

Comment:
Just out of curiosity... do you know why this is multiplied by 0.7?

Reply:
Probably a reserve for overhead, transport protocol, etc.